source/about/mattermost-v10-changelog.md

```diff
@@ -4,6 +4,93 @@
+ - Pre-packaged Playbooks plugin [v2.3.0](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v2.3.0). 
+ - Improved the date and time picker usability across **Custom Status**, **Scheduled Messages**, **Do Not Disturb**, and **Reminder** modals. The entire input area is now clickable, keyboard navigation works properly in time selection menus, and consistent relative date formatting is used throughout. 
+ - Improved the **Permissions** table expand/collapse animation and prevented text from overflowing. 
+ - Added an aria-label to the file preview back/forward buttons. 
+ - Modified the input to have the minimum/maximum length validation work the same as the validation around ``required``, and replaced **Create Team** input with an **Input** component. 
+ - Improved the YouTube video preview user interface. 
+ - Removed the minimum-width for the right-hand side when the window size is <400px. 
+ - Added a status region for the channel filter dropdown. 
+ - Removed the NPS plugin from pre-packaged plugins. 
+
+#### Administration
+ - Signatures are now always validated for pre-packaged plugins. 
+ - Disabled the **Add a license** button when the license is set by an environment variable. 
+ - Improved database connection spikes on user disconnect by processing status updates in batches. 
+ - Improved the efficiency of getting sidebar categories from the database. 
+ - Added a database schema dump to the Support Packet.
+
+#### mmctl Changes
+ - Added ``AuthData`` to mmctl user search output. 
+ - mmctl: Added ``compliance export list`` command. 
+ - mmctl: Added ``compliance export show`` and ``cancel`` commands. 
+ - mmctl: Added ``compliance export download`` command. 
+ - mmctl: Added ``compliance export create`` command. 
+
+### Bug Fixes
+ - Fixed an issue with the onboarding checklist being cut off when on multiple teams. 
+ - Fixed an issue where the **Thread Menu** would not have its actions read by a screen reader. 
+ - Fixed an issue where some users would not get a warning when joining a private channel. 
+ - Fixed an issue with the overflow formatting on the suggestion list. 
+ - Fixed an issue with the **Delete** button aria-label on **User Groups** list modal. 
+ - Fixed some semantic HTML issues with the use of ``header``. 
+ - Fixed various accessibility issues for the **Create Channel** modal. 
+ - Fixed more accessibility issues around search. 
+ - Fixed **Create User Group** modal accessibility issues. 
+ - Fixed mobile view accessibility issues. 
+ - Fix an issue with accents. 
+ - Fixed an issue with rendering of remote user at-mentions in the web app. 
+ - Fixed a potential panic when running the ``mmctl ldap job show`` without the required argument. 
+ - Fixed an issue in the LDAP sync, such that user with an updated attribute were being accidentally added to the groups of another LDAP record, if that LDAP record had a blank ID. 
```

Comment:
Grammar nit:  "such that **users** with an updated attribute were being..." OR "such that user with an updated attribute **was** being..."